### ISSUE TITLE
Make cookiecutter and mgmt command optional

### ISSUE THREAD

**Issue by JulianFeinauer** [timonweb/django-tailwind#134]
Hi,

first off, awesome lib!
But as the management command of the app relies on cookiecutter it brings in all its dependencies which make it quite "heavy" and also introduces a pip module called `text-unidecode` which is dual licensed under GPL / artistic license.
This makes it hard to handle in non-GPL OSS projects OR proprietary software.

I would suggest to make the mgmt command (and thus cookiecutter) optional effectively halving the dependencies.

Here is an output of pipdeptree

```
- django-tailwind [required: ==3.0.1, installed: 3.1.1]
    - cookiecutter [required: >=1.7.3,<2.0.0, installed: 1.7.3]
      - binaryornot [required: >=0.4.4, installed: 0.4.4]
        - chardet [required: >=3.0.2, installed: 4.0.0]
      - click [required: >=7.0, installed: 8.0.4]
      - Jinja2 [required: >=2.7,<4.0.0, installed: 3.0.3]
        - MarkupSafe [required: >=2.0, installed: 2.1.0]
      - jinja2-time [required: >=0.2.0, installed: 0.2.0]
        - arrow [required: Any, installed: 1.2.2]
          - python-dateutil [required: >=2.7.0, installed: 2.8.2]
            - six [required: >=1.5, installed: 1.16.0]
        - jinja2 [required: Any, installed: 3.0.3]
          - MarkupSafe [required: >=2.0, installed: 2.1.0]
      - poyo [required: >=0.5.0, installed: 0.5.0]
      - python-slugify [required: >=4.0.0, installed: 6.1.1]
        - text-unidecode [required: >=1.3, installed: 1.3]
      - requests [required: >=2.23.0, installed: 2.27.1]
        - certifi [required: >=2017.4.17, installed: 2021.10.8]
        - charset-normalizer [required: ~=2.0.0, installed: 2.0.12]
        - idna [required: >=2.5,<4, installed: 3.3]
        - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.8]
      - six [required: >=1.10, installed: 1.16.0]
    - django [required: >=2.2.25, installed: 3.2.12]
      - asgiref [required: >=3.3.2,<4, installed: 3.5.0]
      - pytz [required: Any, installed: 2021.3]
      - sqlparse [required: >=0.2.2, installed: 0.4.2]
    - django-browser-reload [required: >=1.3.0,<2.0.0, installed: 1.3.0]
      - Django [required: >=2.2, installed: 3.2.12]
        - asgiref [required: >=3.3.2,<4, installed: 3.5.0]
        - pytz [required: Any, installed: 2021.3]
        - sqlparse [required: >=0.2.2, installed: 0.4.2]
```

**Comment by NiklasMerz** [comment]
[Cookiecutter has a vulnarability](https://vulners.com/osv/OSV:PYSEC-2022-204) which gets detected by pip-audit right now. Making this big dependency optional sounds like a good idea.

**Comment by timonweb** [comment]
Thanks for the suggestion. It's in `master` already.

**Comment by timonweb** [comment]
Released in 3.3.0
